Sources/Operators.swift

```diff
@@ -16,6 +16,10 @@ import Foundation
 
 // MARK: - <| Operator
 
+precedencegroup Decodative {
+    associativity: left
```

Comment:
Yeah, I would say if it the tests are passing, then we haven't broken anything (fingers crossed 😅) and let's leave it as the default group